modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -0,0 +1,302 @@
+= Partner Manager 2.0 Troubleshooting
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+//Titles of troubleshooting topics must indicate a description about the issue, for example, "Monthly payroll not generating". All or a part of the actual displayed message must be included in the title.
+
+//Include an introduction to explain what the error is about, for example, when you <do this>, you receive <this output> and <this> happens.
+// Try to include as much information as possible about how the error is generated, and where it is generated. Provide tha actual error string, if possible. If the error is generated in a particular component of the product, clearly mention that as well.
+
+
+= Unsuccessful Message Flow Deployment Error
```

Comment:
Should these be H2s (==)?

Reply:
The new template provides these level headings. They don't seem to work for our purposes. If Amit is okay with it, I might bump everything down a level. If I do, it will make some of the 3rd level headings very small.